src/components/nl-design-system-community.scss

```diff
@@ -0,0 +1,13 @@
+@import '@utrecht/components/dist/button/css/index.css';
```

Comment:
I think this file rather belongs in `src/scss` than `src/components` for consistency sake